circle.yml

```diff
@@ -0,0 +1,7 @@
+machine:
+  java:
+    version: oraclejdk8
+
+test:
+  override:
+    - ./gradlew clean check
```

Comment:
Looks good to me!   We may want to "collect" the unit tests etc in CircleCI at some point.